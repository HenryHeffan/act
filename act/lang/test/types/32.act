template<pint N>
interface linear(bool in[N], out[N]);

template<pint N; ptype(linear<N>) p>
defproc foo ()
{
  bool w[N];
  p y;

  y.in = w;
}

foo<4> x;
