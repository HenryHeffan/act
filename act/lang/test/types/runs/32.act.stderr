In expanding foo<4> (32.act:4)
In expanding ::<Global>
FATAL: Ptype `p' used to instantiate `y' is not set
